### PR TITLE
Update Trig Block

### PR DESCRIPTION
**Pull request by Pryme8**
Had to revert the commits, and then resubmit.

### CODE REVIEW

File: src/Materials/Node/Blocks/trigonometryBlock.ts [hunk 1]
```diff
@@ -65,8 +65,8 @@ export class TrigonometryBlock extends NodeMaterialBlock {
     public constructor(name: string) {
         super(name, NodeMaterialBlockTargets.Neutral);
 
-        this.registerInput("input", NodeMaterialBlockConnectionPointTypes.Float);
-        this.registerOutput("output", NodeMaterialBlockConnectionPointTypes.Float);
+        this.registerInput("input", NodeMaterialBlockConnectionPointTypes.AutoDetect);
+        this.registerOutput("output", NodeMaterialBlockConnectionPointTypes.BasedOnInput);
```
Comment (deltakosh) [inline src/Materials/Node/Blocks/trigonometryBlock.ts]: haha you need to connect the output with input still

Comment (Pryme8) [reply src/Materials/Node/Blocks/trigonometryBlock.ts]: can you give me a quick example?

Sorry I am trying to do it from just using the other blocks as an example.

Is this what you are taking about:
this._outputs[0]._typeConnectionSource = this._inputs[0];

**Comment by Pryme8** [comment]
Starting to realize I am really bad at git....

**Comment by Pryme8** [comment]
Whoops sorry did not mean to request the review!

**Comment by deltakosh** [comment]
lol it is good now!

**Pull request merged**